src/data/faq.json

```diff
@@ -50,11 +120,70 @@
                         "indexed": true,
                         "accordion": [
                             {
-                                "title": "Here we will inform you about the functional impacts of the end of development work on the CWA.",
-                                "anchor": "ramp_down_effects",
+                                "title": "Which functions remain in the app after the end of development?",
+                                "anchor": "ramp_down_features",
                                 "active": false,
                                 "textblock": [
-                                    ""
+                                    "Even after the end of the development work on the CWA, you can still access your already scanned digital certificates and you can use the contact diary. However, you will not be able to scan any more certificates or to renew existing certificates. You will also no longer receive notifications about the certificates. Additionally, you can still view statistics of the Pandemic Radar; you find a corresponding link on the status screen of the app.",

```

Comment:
"you find a corresponding link" =>
"you can find a corresponding link"

Reply:
Updated in https://github.com/corona-warn-app/cwa-website/pull/3435/commits/19e4e13371b4215a468ab318569a73ff429312d4, thanks for the correction!